receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper_test.go

```diff
@@ -72,6 +72,30 @@ func TestScrapeMetrics_AllData(t *testing.T) {
 	})
 }
 
+func TestScrapeMetrics_Linux(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		return
+	}
```

Comment:
Alternatively, I could put this test in a separate file that has a build flag so it only runs on Linux, but personally I prefer keeping all the tests together

Reply:
Like how it is.